### DISCUSSION TITLE
Language dictionary not loading

### DISCUSSION THREAD

**Discussion by praneet-ms** [cefsharp/CefSharp#4530] [General]
In only one of our machines, the language dictionary is not loading. Is there any way to debug and find what's going wrong. Also if we download manually and place the language dictionary file in the machine, will it work? Thanks.

**Comment by BuslikDrev** [comment]
Вы каким образом загружаете языковой словарь?

**Comment by amaitland** [comment]
Firstly start by checking the log file to see if there are any errors. See https://github.com/cefsharp/CefSharp/wiki/Trouble-Shooting#log-file

> Also if we download manually and place the language dictionary file in the machine, will it work?

I've never tested this myself.
